### DISCUSSION TITLE
Add a '-' button to delete current Ray screen

### DISCUSSION THREAD

**Discussion by jensolafkoch** [spatie/ray#584] [Ideas]
I always wonder how to easily close a particular screen in Ray when several are open and didn't find a delete button.

I'd expect a '-' button, analogue to the '+' button used to open a new screen.

Maybe that would be a nice addition?

**Comment by AdrianMrn** [comment]
Those buttons are more of a "history", analogous to how you can't skip forward in a browser when having just navigated to a new page.

What exactly do you mean by "when several are open"?

**Reply by jensolafkoch** [reply]
When I've opened a couple of screens with ray()->clearScreen() – which I usually do so that I can faster see where the latest entries are – I think it would be helpful if I could go backwards and then easily close screens at the same nave elements (that is besides the arrows).

For me different Ray screens are more like open files in an editor which I like to be able close easily.
